src/include/partitioning/custom_part_lib.rb

```diff
@@ -672,7 +676,8 @@ def HandlePartWidgetChanges(init, ret, file_systems, old, new)
       end
 
       # set btrfs subvolumes (bnc#872210)
-      if init && new.fetch("mount", "") == "/"
+      # only when requested from the caller (i.e. new partition) (bsc#965279)
+      if init && new.fetch("mount", "") == "/" && force_defaults
```

Comment:
With a smaller or more tested method, I would have changed it to `new_partition`, like I did in my previous PR. But I don't dare in this case.